classes/kernel-dev.yaml

```diff
@@ -0,0 +1,211 @@
+# Development source linux kernel. When built, this recipe packages the \
```

Comment:
Basically this should be part of basement-gnu-linux. Even though we technically have a kernel in the basement repository it's only here to have the libc kernel headers. But on another note:

* why a class and not a multipackage on the kernel::linux recipe?
* recent kernels have the `modules_prepare` build target. Would that help or be usable in your case?
* how is this related to the Ada support? I'd rather keep unrelated things separated in different PRs...

@sixtyfourktec Any more thoughts on this?

Reply:
> Basically this should be part of basement-gnu-linux. Even though we technically have a kernel in the basement repository it's only here to have the libc kernel headers. But on another note:
> 
>     * why a class and not a multipackage on the kernel::linux recipe?
> 
Cause it can than be used in different kernel recipes. (vendor kernels, ...)

>     * recent kernels have the `modules_prepare` build target. Would that help or be usable in your case?
> 
AFAICR this didn't work. But I can't remember why. 

>     * how is this related to the Ada support? I'd rather keep unrelated things separated in different PRs...
> 
The name of the PR is outdated.

I can remove this class and put it in a different layer...